content/200-concepts/100-components/02-prisma-client/053-client-extensions/199-extension-examples.mdx

```diff
@@ -0,0 +1,37 @@
+| [`callback-free-itx`](https://github.com/prisma/prisma-client-extensions/callback-free-itx)       | Adds a method to start interactive transactions without callbacks                           |
+| [`computed-fields`](https://github.com/prisma/prisma-client-extensions/computed-fields)           | Adds virtual / computed fields to result objects                                            |
+| [`input-transformation`](https://github.com/prisma/prisma-client-extensions/input-transformation) | Transforms the input arguments passed to Prisma Client queries to filter the result set     |
+| [`input-validation`](https://github.com/prisma/prisma-client-extensions/input-validation)         | Runs custom validation logic on input arguments passed to mutation methods                  |
+| [`instance-methods`](https://github.com/prisma/prisma-client-extensions/instance-methods)         | Adds Active Record-like methods like `save()` and `delete()` to result objects              |
+| [`json-field-types`](https://github.com/prisma/prisma-client-extensions/json-field-types)         | Uses strongly-typed runtime parsing for data stored in JSON columns                         |
+| [`model-filters`](https://github.com/prisma/prisma-client-extensions/model-filters)               | Adds reusable filters that can composed into complex `where` conditions for a model         |
+| [`obfuscated-fields`](https://github.com/prisma/prisma-client-extensions/obfuscated-fields)       | Prevents sensitive data (e.g. `password` fields) from being included in results             |
+| [`query-logging`](https://github.com/prisma/prisma-client-extensions/query-logging)               | Wraps Prisma Client queries with simple query timing and logging                            |
+| [`readonly-client`](https://github.com/prisma/prisma-client-extensions/readonly-client)           | Creates a client that only allows read operations                                           |
+| [`retry-transactions`](https://github.com/prisma/prisma-client-extensions/retry-transactions)     | Adds a retry mechanism to transactions with exponential backoff and jitter                  |
+| [`row-level-security`](https://github.com/prisma/prisma-client-extensions/row-level-security)     | Uses Postgres row-level security policies to isolate data a multi-tenant application        |
+| [`static-methods`](https://github.com/prisma/prisma-client-extensions/static-methods)             | Adds custom query methods to Prisma Client models                                           |
+| [`transformed-fields`](https://github.com/prisma/prisma-client-extensions/transformed-fields)     | Demonstrates how to use result extensions to transform query results and add i18n to an app |
+| [`exists-method`](https://github.com/prisma/prisma-client-extensions/exists-method)               | Demonstrates how to add an `exists` method to all your models                               |
```

Comment:
these all 404, it should be e.g. `https://github.com/prisma/prisma-client-extensions/tree/main/audit-log-context` instead of `https://github.com/prisma/prisma-client-extensions/audit-log-context`